raft/raftpb/raft.proto

```diff
@@ -43,6 +43,7 @@ enum MessageType {
 	MsgHeartbeat     = 8;
 	MsgHeartbeatResp = 9;
 	MsgUnreachable   = 10;
+	MsgSnapStatus    = 11;
```

Comment:
after i thought this for awhile,

> follower has finished processing the snapshot

If this case, the application should simply not send the snapStauts success msg and raft will wait for the appResp to abort snapshot.

> Whether we allow it over the wire or not

I would say we do not allow it over the wire at this moment. So probably snapStatus is more descriptive?